src/mqtt_client/AtlasMqttClient_callback.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __ATLAS_MQTT_CLIENT_CALLBACK_H__
+#define __ATLAS_MQTT_CLIENT_CALLBACK_H__
+
+#include <mqtt/async_client.h>
+#include <stdlib.h>
+#include <string>
+
+using namespace std;
+
+namespace Atlas
```

Comment:
Please use the existing "atlas" namespace which begins with lower case "a"

Reply:
Done.